File: Cpp/odev2.cpp
#include <iostream>

void sapmaKontrol(float sapma) {

    if (sapma > 1.0f || sapma < -1.0f) {
        std::cout << "ACIL DURUM: Pist Disi! Fren Yap. Sapma: "<< sapma << std::endl;
    }
    else if (sapma >= -0.1f && sapma <= 0.1f) {
        std::cout << "Yolunda: Duz Devam Et. Sapma: "<< sapma << std::endl;
    }
    else if (sapma > 0.1f) {
        std::cout << "Saga Kaydin: Sola Direksiyon Kir. Sapma: "<< sapma<< std::endl;
    }else { // sapma < -0.1
        std::cout << "Sola Kaydin: Saga Direksiyon Kir. Sapma: "<< sapma << std::endl;
    }
}

int main() {
    float sapmaDegeri;

    std::cout << "Serit merkezinden sapma degerini girin: ";
    std::cin >> sapmaDegeri;

    sapmaKontrol(sapmaDegeri);

    return 0;
}

File: python/odev2.py
def sapma_kontrol(sapma: float):
    if sapma > 1.0 or sapma < -1.0:
        print(f"ACIL DURUM: Pist Disi! Fren Yap. (Sapma: {sapma:.2f} m)")
    elif -0.1 <= sapma <= 0.1:
        print(f"Yolunda: Duz Devam Et. (Sapma: {sapma:.2f} m)")
    elif sapma > 0.1:
        print(f"Saga Kaydin: Sola Direksiyon Kir. (Sapma: {sapma:.2f} m)")
    else:
        print(f"Sola Kaydin: Saga Direksiyon Kir. (Sapma: {sapma:.2f} m)")


sapma_degeri = float(input("Metre cinsinden sapma degerini giriniz: "))
sapma_kontrol(sapma_degeri)
